tracing-subscriber/src/registry/fmt_layer.rs

```diff
@@ -0,0 +1,197 @@
+use crate::{
+    fmt::{format, FormatEvent, FormatFields, MakeWriter},
+    layer::{Context, Layer},
+    registry::{Extensions, LookupMetadata, LookupSpan, Registry, SpanData, SpanRef},
+};
+use ansi_term::{Color, Style};
+use humantime;
+use std::{
+    any::type_name,
+    fmt::{self, Write as _},
+    io::{self, Write},
+    marker::PhantomData,
+    time::SystemTime,
+};
+use tracing_core::{
+    field::{Field, Visit},
+    span::{Attributes, Id},
+    Event, Level, Subscriber,
+};
+
+pub struct FmtLayer<
+    S = Registry,
+    N = format::DefaultFields,
+    E = format::Format<format::Full>,
+    W = fn() -> io::Stdout,
+> {
+    is_interested: Box<dyn Fn(&Event<'_>) -> bool + Send + Sync + 'static>,
+    inner: PhantomData<S>,
```

Comment:
nit: PhantomData field should probably begin with an `_` (as it will never be used)
```suggestion
    _inner: PhantomData<S>,
```

Reply:
done, thanks!